src/content/developers/docs/scaling/index.md

```diff
@@ -0,0 +1,90 @@
+## Prerequisites {#prerequisites}
+
+You should have a good understanding of all the foundational topics. Implementing scaling solutions is advanced as the technology is less battle-tested, and continues to be researched and developed.
+
+## On-Chain Scaling {#on-chain-scaling}
+
+This method of scaling requires changes to the existing layer 1 [mainnet](/glossary/#mainnet). Sharding is currently the main focus for this method of scaling.
+
+### Sharding {#sharding}
+
+Sharding is the process of splitting a database horizontally to spread the load. In an Ethereum context, sharding will reduce network congestion and increase transactions per second by creating new chains, known as “shards.” This will also lighten the load for each validator who will no longer be required to process the entirety of all transactions across the network.
+
+Learn more about [sharding](/eth2/shard-chains/).
+
+## Off-Chain Scaling {#off-chain-scaling}
+
+Off-chain solutions are designed separate from layer 1 mainnet. Some solutions, known as "layer 2" solutions, derive their security directly from layer 1 Ethereum consensus, such as rollups or channels. Other solutions involve the creation of new chains in various forms that derive their security separately from mainnet, such as [sidechains](#sidechains) or [plasma](#plasma) chains. These solutions communicate with mainnet, but derive their security differently to obtain a variety of goals.
+
+### Layer 2 Scaling {#layer-2-scaling}
+
+This category of off-chain solutions derives its security from mainnet Ethereum.
+
+Most layer 2 solutions are centered around a server or cluster of servers, each of which may be referred to as a node, validator, operator, sequencer, block producer, or similar term. Depending on the implementation, these layer 2 nodes may be run by the businesses or entities that use them, or by a 3rd party operator, or by a large group of individuals (similar to mainnet). Generally speaking, transactions are submitted to these layer 2 nodes instead of being submitted directly to layer 1 (mainnet); the layer 2 instance then batches them into groups before anchoring them to layer 1, after which they are secured by layer 1 and cannot be altered. The details of how this is done vary significantly between different layer 2 technologies and implementations.
+
+A specific layer 2 instance may be open and shared by many applications, or may be deployed by one company and dedicated to supporting only their application.
+
+#### Rollups {#rollups}
+
+Rollups perform transaction execution outside layer 1 and then the data is posted to layer 1 where consensus is reached. As transaction data is included in layer 1 blocks, this allows rollups to be secured by native Ethereum security.
+
+- [ZK rollups](/developers/docs/scaling/layer-2/#zk-rollups)
+- [Optimistic rollups](/developers/docs/scaling/layer-2/#optimistic-rollups)
+
+Learn more about [rollups](/developers/docs/scaling/layer-2/).
+
+#### Channels {#channels}
```

Comment:
Should we go with "channels" or "state channels"? I feel like "state channels" is more descriptive.